plugins/modules/source_control/github/github_issue.py

```diff
@@ -15,36 +17,42 @@
 }
 
 
-DOCUMENTATION = '''
+DOCUMENTATION = r'''
 module: github_issue
 short_description: View GitHub issue.
 description:
     - View GitHub issue for a given repository and organization.
+version_added: "2.4"
 options:
   repo:
     description:
       - Name of repository from which issue needs to be retrieved.
     required: true
+    type: str
   organization:
     description:
       - Name of the GitHub organization in which the repository is hosted.
     required: true
+    type: str
   issue:
     description:
       - Issue number for which information is required.
     required: true
+    type: int
   action:
     description:
         - Get various details about issue depending upon action specified.
     default: 'get_status'
     choices:
         - 'get_status'
+    type: str
+extends_documentation_fragment: community.general.github
 author:
     - Abhijeet Kasurde (@Akasurde)
 '''
 
 RETURN = '''
-get_status:
+issue_state:
```

Comment:
Also here: this breaks backwards compatibility.

Reply:
`issue_state` is correct return. I think `get_status` was added adverently.